app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/address/disposal_price.scala.html

```diff
@@ -73,20 +74,27 @@
 
     @caption(addressCaption)
 
+    <h1 class="govuk-heading-xl">@title</h1>
+
+    <p class ="govuk-body">@messages(s"$key$userKey.helpText")</p>
+
+    @details(
+        messages(s"$key.link"),
+        Html(
+            s"""
+          | <p class="govuk-body">${messages(s"$key.details.p1")}</p>
+          |""".stripMargin
+        )
+    )
+
+    <h2 class="govuk-fieldset__legend govuk-fieldset__legend--m">@h2</h2>
```

Comment:
I checked with Adam Liptrot, we do not want to be using a h2 for the input label